distribution/scripts/jmeter/start-jmeter-test.sh

```diff
@@ -346,6 +333,28 @@ function test_scenarios() {
             if [ $skip = true ]; then
                 continue
             fi
+
+            export scenario_name=${scenario[name]}
+            export scenario_flags=${scenario[netty_options]}
+            
+            # Create Docker images and push to ECR
+            $SCRIPTS_DIR/netty/make-netty-image.sh $scenario_flags
+            $SCRIPTS_DIR/ballerina/make-ballerina-image.sh -t $scenario_name
+
+            # Create ECS cluster
+            $SCRIPTS_DIR/cloudformation/ecs-cfn.sh -t $scenario_name
+
+            # Take Ballerina task IP
+            aws cloudformation wait stack-create-complete --stack-name ecs-stack
+
+            taskid=$(aws ecs list-tasks --cluster ballerina-performance-test --service-name ballerina-service --query ["taskArns"][0][0] --output text)
+            hostname=$(aws ecs describe-tasks --cluster ballerina-performance-test --tasks $taskid --query 'tasks[*].attachments[].details[].value | [4]' --output text)
```

Comment:
At which point we pass this `hostname` value to JMeter?

Reply:
By calling "before_execute_test_scenario" (ballerina-test-config.sh) at line 382 .